src/Contracts/BatchesResults.php

```diff
@@ -0,0 +1,94 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Meilisearch\Contracts;
+
+class BatchesResults extends Data
+{
+    /**
+     * @var non-negative-int
+     */
+    private int $next;
+
+    /**
+     * @var non-negative-int
+     */
+    private int $limit;
+
+    /**
+     * @var non-negative-int
+     */
+    private int $from;
+
+    /**
+     * @var non-negative-int
+     */
+    private int $total;
+
+    public function __construct(array $params)
+    {
+        parent::__construct($params['results'] ?? []);
```

Comment:
Are there any cases when results will not be set?

Reply:
As far as I understand, the API should always return a `results` array. 

But I saw in the `TasksResults` class that there was a check for the field being missing. So I used the same logic.

Reply:
So it's a dead code then :slightly_smiling_face:

Reply:
I updated this line ✅